spec/controllers/reservations_controller_spec.rb

```diff
@@ -29,8 +30,45 @@
     end
   end
 
+  describe "DELETE #destroy" do
+    context "when user is logged in" do
+      before do
+        sign_in_user
+        delete_destroy(wishlist.id, item.id, reservation.id)
```

Comment:
I'm not sure, but in other controllers - WishlistsController and ItemsController - we specified relationships.